src/uct/api/uct.h

```diff
@@ -1413,19 +1413,22 @@ struct uct_tag_context {
     /**
      * Tag processing is completed by the transport.
      *
-     * @param [in]  self    Pointer to relevant context structure, which was
-     *                      initially passed to @ref uct_iface_tag_recv_zcopy.
-     * @param [in]  stag    Tag from sender.
-     * @param [in]  imm     Immediate data from sender. For rendezvous, it's always 0.
-     * @param [in]  length  Completed length.
+     * @param [in]  self        Pointer to relevant context structure, which was
+     *                          initially passed to @ref uct_iface_tag_recv_zcopy.
+     * @param [in]  stag        Tag from sender.
+     * @param [in]  imm         Immediate data from sender. For rendezvous, it's always 0.
+     * @param [in]  length      Completed length.
+     * @param [in]  inline_data If non-null, points to a temporary buffer which contains
+                                the received data. In this case the received data was not
+                                placed directly in the receive buffer.
```

Comment:
Maybe add - user is responsible to copy out the data before completion of the callback, otherwise the inline data is released ?

Reply:
@shamisp can you please check now?

Reply:
Looks good !